engine/render/renderer/swapchain/Swapchain.cpp

```diff
@@ -177,14 +166,15 @@ void SwapChain::CreateSwapChain()
 
     CC_ASSERT(
         vkCreateSwapchainKHR(device.Device(), &createInfo, nullptr, OUT & swapChain) == VK_SUCCESS,
-        "Failed to create Swapchain!");
+        "Failed to create Swapchain!")
```

Comment:
Please rewrite `CC_ASSERT` to **require** a semicolon at the end to keep this consistent with `CC_LOG`.

Reply:
Done. I've created a separate function which the macro wraps.

Reply:
Function:

```cpp
inline void Assert(bool expr, const char* file, int line, const char* message)
{
    if (expr)
    {}
    else
    {
        REPORT_ASSERT_FAILURE(expr, file, line, message);
        EXIT_APP
    }
}
```

Macro: 

```cpp
#define CC_ASSERT(expr, message) Siege::Assert(expr, __FILE__, __LINE__, message)
```